server/webhook.go

```diff
@@ -0,0 +1,213 @@
+package main
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"crypto/subtle"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"math"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+	"github.com/pkg/errors"
+
+	"github.com/mattermost/mattermost-plugin-zoom/server/zoom"
+)
+
+func (p *Plugin) handleWebhook(w http.ResponseWriter, r *http.Request) {
+	if !p.verifyWebhookSecret(r) {
+		p.API.LogWarn("Could not verify webhook secret")
+		http.Error(w, "Not authorized", http.StatusUnauthorized)
+		return
+	}
+
+	if !strings.Contains(r.Header.Get("Content-Type"), "application/json") {
+		res := fmt.Sprintf("Expected Content-Type 'application/json' for webhook request, received '%s'.", r.Header.Get("Content-Type"))
+		p.API.LogWarn(res)
```

Comment:
`LogError` should ideally be reserved for errors that we think the admin should definitely know or may be able to fix. IMO `LogWarn` is basically "something went wrong, but it may be too low-level to remedy as an admin"